docs/source/user-guide/tasks/manage-virtual.rst

```diff
@@ -68,3 +69,5 @@ using an environment variable.  Supported variables include:
 
 * ``CONDA_OVERRIDE_CUDA`` - CUDA version number or set to ``""`` for no CUDA
   detected.
+* ``CONDA_OVERRIDE_GLIBC`` - CUDA version number or set to ``""`` for no GLIBC.
```

Comment:
```suggestion
* ``CONDA_OVERRIDE_GLIBC`` - GLIBC version number or set to ``""`` for no GLIBC.
```

Reply:
Should `""` be interpreted as "no GLIBC" in this case? Admittedly that was true of CUDA, but perhaps this is a different story with GLIBC.